.travis.yml

```diff
@@ -1,25 +1,85 @@
-dist: xenial
+dist: bionic
 language: python
 python:
   - 3.6
 git:
+  ## depth set to false overrides travis default behavior to use shallow clones
+  ## with depth 50 that can cause issues
```

Comment:
Question on style: Any particular reason to use double hashes for comments?

Reply:
I've grown to like it a lot from GitLab's Helm chart yaml files, but I guess it makes less sense in a bash script. It separates the example comments from the descriptive comments. Like here for example

```yaml
    ## doc/charts/globals.md#ldap
    ldap:
      servers: {}
      ## 'main' is the GitLab 'provider ID' of this LDAP server
      # main:
      #   label: 'LDAP'
      #   host: '_your_ldap_server'
      #   port: 636
      #   uid: 'sAMAccountName'
      #   bind_dn: '_the_full_dn_of_the_user_you_will_bind_with'
      #   password:
      #     secret: _the_secret_containing_your_ldap_password
      #     key: _the_key_which_holds_your_ldap_password
      #   encryption: 'plain'
```

Reply:
Thanks for the explanation @consideRatio. It makes sense for YAML as in the example above. For Python, it probably makes sense to use the double # sparingly. I'm not sure about bash.